test/LdapTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
```

Comment:
Good point! I was thinking of moving it to an LdapDeleteAttributeTest-class as All other tests are splitted in that way. So I can later (as soon as Travis-tests can use an LDAP) also add an online-test in the same class. That way all tests regarding this feature are grouped together in one place.

Or do you think moving them into the offlineTest-class is the better alternative?